packages/delegation-process/src/services/readModelService.ts

```diff
@@ -105,13 +105,14 @@ export function readModelServiceBuilder(
     async getDelegation(
       delegations: DelegationCollection,
       filter: Filter<WithId<WithMetadata<DelegationReadModel>>>
-    ): Promise<Delegation | undefined> {
+    ): Promise<WithMetadata<Delegation> | undefined> {
       const data = await delegations.findOne(filter, {
         projection: { data: true, metadata: true },
       });
```

Comment:
Oops...This slipped in a previous PR, but there should be a sort by `createdAt` in the `findOne`, because the delegated (the one who receives the delegate) can have multiple delegations (ex. 2 revoked, and the last one active)

Reply:
Make sense to add a sorting on metadata version.